Product/Production/Services/CORE_X12DocumentSubmissionCore/src/main/java/gov/hhs/fha/nhinc/corex12/docsubmission/audit/transform/CORE_X12AuditDataTransform.java

```diff
@@ -358,65 +352,6 @@ protected boolean areRequiredCOREX12BatchFieldsNull(Object msg, AssertionType as
         if (oXDR.areRequiredUserTypeFieldsNull(assertion)) {
```

Comment:
I don't think we need the assertion required fields checked either.  If those are null, we still should pass it.